ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -103,6 +90,8 @@ object AuthServiceJWTCodec {
   // ------------------------------------------------------------------------------------------------------------------
   // OpenID Connect (OIDC) namespace for custom JWT claims
   final val oidcNamespace: String = "https://daml.com/ledger-api"
+  // Unique scope for standard tokens, following the pattern of https://developers.google.com/identity/protocols/oauth2/scopes
+  final val scopeLedgerApiFull: String = "https://daml.com/ledger-api.full_control"
```

Comment:
Note: more work required to figure out the right name here. Most scopes used by Google use a URL of the form `https://www.googleapis.com/auth/<api-name>` and they use `api-name` for full access and `api-name.read_only` for read-only access. The address at `https://www.googleapis.com/auth/<api-name>` also serves `<api-name>` if and only if that is a defined scope name.

So for Daml that would mean:
```
https://daml.com/auth/ledger-api                    for full control over the ledger api of a given user including admin functionality
https://daml.com/auth/ledger-api.read_only          for read only access to the ledger api calls of a given user, if we decide to introduce this
https://daml.com/auth/<new-api>                     space for introducing new api's in the future
```

So, I'd suggest to go with `https://daml.com/auth/ledger-api ` for now. What do you think @da-tanabe @mziolekda @cocreature @bame-da @nycnewman, are you OK with that proposal ?

Reply:
Sounds good to me.

Reply:
An alternative would be to call the scope `https://daml.com/auth/participant-node` to acknowledge that the user's access to all APIs of a participant node is being delegated.

Reply:
I expect `ledger-api` is clearer to our users than `participant-node`.

Reply:
I'll move to using `daml_ledger_api` as the scope for now, as that will definitely be accepted given it is the common naming scheme within the OAuth 2.0 and OIDC standards. This way we maximize compatiblity, while having a high likelyhood that there are no such tokens already out there.